src/components/app/Nav.svelte

```diff
@@ -13,6 +31,11 @@
   </div>
   <div class="right">
     <slot name="right" />
+    <select value={user?.activeRole} class="st-select" on:change={changeRole}>
```

Comment:
I'm wondering if we should hide this in the case `allowedRoles` just has a single value?

Reply:
@camargo Pushed a slight tweak to handle the single case